<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>error_dialog</class>
 <widget class="QDialog" name="error_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>140</width>
    <height>191</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>140</width>
    <height>191</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ошибка!</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ОШИБКА!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="error_text"/>
    </item>
    <item>
     <widget class="QPushButton" name="error_button">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
